models/examineSearchInfo.js

```diff
@@ -50,3 +57,46 @@ const getWorld = (id) => { // TODO: use World exd
     if (id ===  97) return "Ragnarok";
     if (id ===  99) return "Sargatanas";
 };
+const getJob = (id) => {
+    if (id ===  1) return ""; // ? Unknow
+    if (id ===  2) return "GLD";
+    if (id ===  3) return "PGL";
+    if (id ===  4) return "MRD";
+    if (id ===  5) return "LNC";
+    if (id ===  6) return "ARC";
+    if (id ===  7) return "CNJ";
+    if (id ===  8) return "THM";
+    if (id ===  9) return "CRP";
+    if (id ===  10) return "BSM";
+    if (id ===  11) return "ARM";
+    if (id ===  12) return "GSM";
+    if (id ===  13) return "LTW";
+    if (id ===  14) return "WVR";
+    if (id ===  15) return "ALC";
+    if (id ===  16) return "CUL";
+    if (id ===  17) return "MIN";
+    if (id ===  18) return "BTN";
+    if (id ===  19) return "FSH";
+    if (id ===  20) return "PLD";
+    if (id ===  21) return "MNK";
+    if (id ===  22) return "WAR";
+    if (id ===  23) return "DRG";
+    if (id ===  24) return "BRD";
+    if (id ===  25) return "WHM";
+    if (id ===  26) return "BLM";
+    if (id ===  27) return "ACN";
+    if (id ===  28) return "SCH"; // Need confirm
+    if (id ===  29) return "SMN"; // Need confirm
+    if (id ===  30) return "ROG";
+    if (id ===  31) return "NIN";
+    if (id ===  32) return "MCH";
+    if (id ===  33) return "DRK";
+    if (id ===  34) return "AST";
+    if (id ===  35) return "SAM";
+    if (id ===  36) return "RDM";
+    if (id ===  37) return "BLU";
+    if (id ===  38) return "GNB";
+
+    // DNC is missing ?
```

Comment:
DNC is ID 38, these all need to be shifted back one so that DNC can be included.

Reply:
From my own data, for reference: `37,0,80,0,38,0,70,0`. I have GNB (37) at 80, and DNC (38) at 70.